packages/itwinui-react/src/react-table/react-table.ts

```diff
@@ -1193,9 +1193,52 @@ export interface SortingRule<D> {
 
 //#endregion
 
+type ActionTypes =
```

Comment:
Can we split this into two types (built-in actions and custom actions) and define `ActionType` using a union `BuiltInAction | CustomAction | AnyString`?

It doesn't change anything for the user, but would help with our own maintainability.

also nit: the type name should be singular, not plural, since it is equal to *one* of these values.